apisix/discovery/consul_kv.lua

```diff
@@ -230,6 +241,73 @@ local function update_application(server_name_prefix, data)
 end
 
 
+local function read_dump_srvs()
+    local data, err = util.read_file(dump_params.path)
+    if not data then
+        log.warn("read dump file get error: ", err)
```

Comment:
If we don't care about the existence of the dump file, maybe we can use a lower log level when we don't find the file?

Reply:
No, haha :))
Use warn level is better than info level here for unexpected event.

Reply:
So you suggest to prepare the dump file before staring APISIX to avoid the unexpected event (file not found)?

Reply:
No, we don't prepare the dump file.
Maybe the log level is `notice` would be suitable.